siddhi-extensions/geo/geo-proxymity-operation/src/test/java/org/wso2/cep/geo/proximity/GeoProximityTestCase.java

```diff
@@ -0,0 +1,119 @@
+package org.wso2.cep.geo.proximity;
+
+import org.wso2.siddhi.core.SiddhiManager;
+import org.wso2.siddhi.core.config.SiddhiConfiguration;
+import org.wso2.siddhi.core.event.Event;
+import org.wso2.siddhi.core.query.output.callback.QueryCallback;
+import org.wso2.siddhi.core.stream.input.InputHandler;
+import org.wso2.siddhi.core.util.EventPrinter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class GeoProximityTestCase {
+    public static void main(String[] args) throws Exception {
+
+        //from anurudhdha
+        SiddhiConfiguration conf = new SiddhiConfiguration();
+        List<Class> classList = new ArrayList<Class>();
+        classList.add(GeoProximity.class);
+        conf.setSiddhiExtensions(classList);
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+        siddhiManager.getSiddhiContext().setSiddhiExtensions(classList);
+
+				/*siddhiManager
```

Comment:
Thanks @lahirue , This code has been refactored and no longer available in the repository, Since could not fix the issue.